tests/test_repocard_data.py

```diff
@@ -91,6 +91,8 @@ def test_model_index_to_eval_results(self):
                             {
                                 "type": "acc",
                                 "value": 0.9,
+                                "verified": True,
```

Comment:
I found it useful to add this info to the unit test to be sure that these new fields are behaving as expected